test/openshift/application.test.ts

```diff
@@ -22,8 +22,8 @@ suite('Openshift/Application', () => {
     let sandbox: sinon.SinonSandbox;
     let execStub: sinon.SinonStub;
     let getProjectNamesStub: sinon.SinonStub;
-    const projectItem = new TestItem(null, 'project');
-    const appItem = new TestItem(projectItem, 'app');
+    const projectItem: OpenShiftObject = new TestItem(null, 'project');
```

Comment:
No need to declare type for direct assignment, it is going to be inferred during the compilation.

Reply:
All the changes below have the same problems.